pkg/retry/retry_test.go

```diff
@@ -137,6 +137,7 @@ func TestRetrier_RetryWithoutBackoff(t *testing.T) {
 			rtr := retry.New(tc.cfg)
 			_ = rtr.Run(context.TODO(), tr.Execute)
 
+			// assert.Len(t, tr.waitTimes, len(tc.expectedWaitTimes))
```

Comment:
this looks like unused

Reply:
This check is internally used by the assert package, this comment is just for the understanding that if we pass not retryable error it will not it will give an empty response. And test case will fail. 
https://github.com/stretchr/testify/blob/master/assert/assertions.go#L1517